RNWCS/local-cli/rnpm/windows/src/windows.js

```diff
@@ -21,98 +20,75 @@ const REACT_NATIVE_WINDOWS_GENERATE_PATH = function() {
   );
 };
 
-const RNWCPP_GENERATE_PATH = function() {
-  return path.resolve(
-    process.cwd(),
-    'windows',
-    'lib',
-    'react-native-windows',
-    'RNWCPP',
-    'local-cli',
-    'generate-windows.js'
+/**
+ * Returns latest version of react-native-windows where version contains 'vnext'
+ * @param {'yarn'|'npm'} pkgmgr
+ * @param {boolean} isYarn
+ */
+const getLatestVnextVersion = function (pkgmgr, isYarn) {
```

Comment:
For npm you can call:
`npm info react-native-windows@vnext version` -- And that should give you the version you want.

For yarn, I think you want:
`JSON.parse(exec('yarn info react-native-windows --silent'))['dist-tags'].vnext;`